mongodbatlas/resource_mongodbatlas_cluster_test.go

```diff
@@ -1410,6 +1412,49 @@ func TestAccClusterRSCluster_basicAWS_PausedToUnpaused(t *testing.T) {
 	})
 }
 
+func TestAccClusterRSCluster_withDefaultBiConnectorAndAdvancedConfiguration_maintainsBackwardCompatibility(t *testing.T) {
+	var (
+		cluster      matlas.Cluster
+		resourceName = "mongodbatlas_cluster.test"
+		orgID        = os.Getenv("MONGODB_ATLAS_ORG_ID")
+		projectName  = acctest.RandomWithPrefix("test-acc")
+		name         = fmt.Sprintf("test-acc-%s", acctest.RandString(10))
+		config       = testAccMongoDBAtlasClusterConfigAWS(orgID, projectName, name, true, true)
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheckBasic(t) },
+		CheckDestroy: testAccCheckMongoDBAtlasClusterDestroy,
+		Steps: []resource.TestStep{
+			{
+				ExternalProviders: map[string]resource.ExternalProvider{
+					"mongodbatlas": {
+						VersionConstraint: "1.11.0",
```

Comment:
wondering if we want to respect the ```TestAccMigration*``` prefix convention, and add a comment specifying why we leave a fixed version here vs using `lastVersionConstraint` like in our other migration tests (https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1565). I understand we want to test specifically the upgrade from 1.11.x due to terraform framework changes in the 1.12.0 release and ensure this schema change has no impact.

Reply:
I would probably defer that to @maastha since her change hasn't still been deployed. @maastha  you can take a note about this test and include it as a migration test. I left a comment in your PR though that I am not sure why this is called "migration.

@AgustinBettati if you are ok let's leave it like that, I think the name of the test should be enough self-explanatory

Reply:
Yes no need to block this PR, the test as is serving its purpose.